### PR TITLE
kubernetes: support getting kubernetes cluster details with projects command

### PR DESCRIPTION
**Pull request by MorrisLaw**
Currently, we can `doctl projects resources assign` and `doctl projects resources list` the `Kubernetes` resource. But we can't run the `doctl projects resources get do:doks:<cluster-uuid>` command to get the details of the resource. We receive this error instead:
```go
Error: "kubernetes" is an invalid resource type, consult the documentation
```

- This PR attempts to add the `get` command so that Kubernetes cluster details are returned are returned just like when we run `doctl kubernetes cluster get <cluster-uuid>`.

**Note for reviewers:** marking this as WIP for now until I can test it locally.

### CODE REVIEW

File: commands/projects.go [hunk 1]
```diff
@@ -265,6 +265,8 @@ func RunProjectResourcesGet(c *CmdConfig) error {
 		return RunDomainGet(c)
 	case "volume":
 		return RunVolumeGet(c)
+	case "kubernetes":
+		return RunKubernetesClusterGet(c)
```
Comment (andrewsomething) [inline commands/projects.go:269]:
```suggestion
		k8sCmdService := kubernetesCommandService()
		return k8sCmdService.RunKubernetesClusterGet(c)
```

**Review by ChiefMateStarbuck — COMMENTED** [review]
👍🏾

**Comment by MorrisLaw** [comment]
rebased

**Comment by ChiefMateStarbuck** [comment]
The `get cluster` function uses the `list cluster` API endpoint and searches for the specific cluster by id or name. These added commits address this functionality within the `project test get` file.

**Review by danaelhe — APPROVED** [review]
Thank you for your contributions!

**Pull request merged**